src/main/java/org/yomigae/UIFixtureType.java

```diff
@@ -15,9 +15,9 @@
   public UIFixtureType(final LXStudio.UI ui, LX lx) {
     super(ui, title, filename);
     this.lx = lx;
-    registerDiscreteParameter("Type", 1, 1, Output.LightType.OPPSKPAR.getValue() + 1);
+    registerDiscreteParameter("Type", 1, 1, Output.LightType.PRODPAR.getValue() + 1);
```

Comment:
What is the purpose of choosing fixture type?

Reply:
This was used by Tracy's previous Output layer.  I think I only made this change to get everything to compile.